README.md

```diff
@@ -7,9 +7,48 @@ Covata Delta provides an easy to use framework for sharing secrets across networ
 
 ## Requirements
 
-- Python 2.7 +
-- Python 3.3 +
+- Python 2.7 + or 3.3 +
+- pip 9.0.1 +
+- virtualenv: `sudo pip install virtualenv`
+- pybuilder: `pip install pybuilder`
 
+```bash
+sudo pip install virtualenv
+virtualenv venv
+source venv/bin/activate
+pip install pybuilder 
+```
+
+## Quick Start
+
+### Building the project
+
+```bash
+# 1. Check out the project
```

Comment:
Move these inline comments outside of the bash block (to be consistent with the other steps where we use bullet points).